brainstormforce/ultimate-addons-for-gutenberg
# Fix: Info box - Icon Left margin issue and spacing label.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/4a70e27c-773e-4536-992c-78f25fc2fada?redirect=true&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop Info-box check icon/image left margin is set to 10.
- Change preset to 2 or 3 and check if it changes to 0 or not.
- Check spacing label is adjusted properly below each value.

### Checklist:
